mbutil/util.py

```diff
@@ -207,6 +207,8 @@ def disk_to_mbtiles(directory_path, mbtiles_file, **kwargs):
                 y = flip_y(z, int(row_dir.replace("R", ""), 16))
             elif kwargs.get("scheme") == 'gwc':
                 pass
+            elif kwargs.get("scheme") == 'zyx':
+                y = flip_y(int(z), int(row_dir))
```

Comment:
@andrewharvey I'd like to see a test added to make sure that this works and continues to work in the future.

Reply:
Agreed, I've added a test.